### PR TITLE
Add endpoints to modify and delete stands

### PR DESCRIPTION
**Pull request by AxonC**

### CODE REVIEW

File: app/Http/Controllers/Admin/StandAdminController.php [hunk 1]
```diff
@@ -84,30 +88,104 @@ public function createNewStand(Airfield $airfield, StandCreateRequest $request):
...
+
+    /**
+     * Modify a stand which is contained within a given airfield.
+     *
+     * @param Airfield $airfield
+     * @param Stand $stand
+     * @param StandCreateRequest $request
+     * @return JsonResponse
+     */
+    public function modifyStand(Airfield $airfield, Stand $stand, StandCreateRequest $request) : JsonResponse //NOSONAR
```
Comment (AndyTWF) [inline app/Http/Controllers/Admin/StandAdminController.php:112]: Why NOSONAR?

Comment (AndyTWF) [inline app/Http/Controllers/Admin/StandAdminController.php:112]: Should we rename StandCreateRequest to something more generic now it's being used for both update and create?

File: app/Http/Controllers/Admin/StandAdminController.php [hunk 2]
```diff
@@ -84,30 +88,104 @@ public function createNewStand(Airfield $airfield, StandCreateRequest $request):
...
+    {
+        if ($stand->airfield_id != $airfield->id) {
+            return response()->json(self::STAND_NOT_IN_AIRFIELD_ERROR, 404);
+        }
+
+        $validatorsInUse = $airfield->stands->pluck('identifier');
```
Comment (AndyTWF) [inline app/Http/Controllers/Admin/StandAdminController.php:118]: Would this prevent your from you to re-saving the stand with the same identifier, do we need to do a check to the effect of "not if the stand we're updating is the current one"?

**Review by AndyTWF — CHANGES_REQUESTED** [review]

File: app/Http/Controllers/Admin/StandAdminController.php (hunk 1 above)
Comment (AxonC) [reply app/Http/Controllers/Admin/StandAdminController.php:112]: It doesn't like more than 3 returns :(

File: app/Http/Controllers/Admin/StandAdminController.php (hunk 2 above)
Comment (AxonC) [reply app/Http/Controllers/Admin/StandAdminController.php:118]: Good catch, I've amended with tests :)

File: app/Http/Controllers/Admin/StandAdminController.php (hunk 1 above)
Comment (AxonC) [reply app/Http/Controllers/Admin/StandAdminController.php]: Done!

**Review by AndyTWF — APPROVED** [review]

**Pull request merged**